RateLimiter.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
don't name files in pascal case; snake case is preferred

Reply:
renamed the file